src/client/pages/SignIn.tsx

```diff
@@ -174,25 +178,28 @@ export const SignIn = ({
     executeCaptcha();
   };
 
+  const tabs: TabType[] = [
```

Comment:
Suggestion: Instead of pushing to the tabs array at runtime, we could declare both possible tab list states more concretely with something like this:

```
const signInTab = {
  displayText: 'Sign in',
  queryParams: queryParams,
  linkTo: '/signin',
  isActive: true,
}

const registerTab = {
  displayText: 'Register',
  queryParams: queryParams,
  linkTo: '/register',
  isActive: false,
}

const tabs = displayRegisterTab ? [signInTab, registerTab] : [signInTab];
```

Reply:
Will this be beneficial for memory reasons or something else?

Reply:
I was thinking more in terms of readability, both possible list states are clearly defined here, with named vars and no need to push